packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -291,8 +294,8 @@ export class RunningSummarizer implements IDisposable {
                 const summarizeProps: ITelemetryProperties = {
```

Comment:
nit: The comment before the "for" loop should be removed because in your previous change you removed `retryNumber` (it won't let me comment in that line).

Reply:
Also, nit: the variable `retriesPerPhase` should be called `retryPerPhase` to keep it consistent with the telemetry property rename.,